docs/book/advanced_guide/guide-aws-gcp-azure.md

```diff
@@ -0,0 +1,246 @@
+  it to talk to your EKS cluster using the following command. Make sure you have
+  the `aws` cli set up first.
+
+  ```powershell
+  aws eks --region REGION update-kubeconfig --name CLUSTER_NAME
+  ```
+* [Install](https://www.kubeflow.org/docs/components/pipelines/installation/standalone-deployment/#deploying-kubeflow-pipelines)
+  Kubeflow Pipelines onto your cluster.
+  {% endtab %}
+
+{% tab title="GCP" %}
+
+* Have an existing
+  GCP [GKE cluster](https://cloud.google.com/kubernetes-engine/docs/quickstart)
+  set up.
+* Download and [install](https://kubernetes.io/docs/tasks/tools/) `kubectl`
+  and [configure](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl)
+  it to talk to your GKE cluster using the following command. Make sure you have
+  the Google Cloud CLI set up first.
+
+  ```powershell
+  gcloud container clusters get-credentials CLUSTER_NAME
+  ```
+* [Install](https://www.kubeflow.org/docs/distributions/gke/deploy/overview/)
+  Kubeflow Pipelines onto your cluster.
+{% endtab %}
+
+{% tab title="Azure" %}
+
+* Have an
+  existing [AKS cluster](https://azure.microsoft.com/en-in/services/kubernetes-service/#documentation)
+  set up.
+* Download and [install](https://kubernetes.io/docs/tasks/tools/) `kubectl` and
+  it to talk to your AKS cluster using the following command. Make sure you have
+  the `az` cli set up first.
+
+  ```powershell
+  az aks get-credentials --resource-group RESOURCE_GROUP --name CLUSTER_NAME
+  ```
+* [Install](https://www.kubeflow.org/docs/components/pipelines/installation/standalone-deployment/#deploying-kubeflow-pipelines)
+  Kubeflow Pipelines onto your cluster.
+
+> Since Kubernetes v1.19, AKS has shifted
+> to [`containerd`](https://docs.microsoft.com/en-us/azure/aks/cluster-configuration#container-runtime-configuration)
+> . However, the workflow controller installed with the Kubeflow installation
+> has `Docker` set as the default runtime. In order to make your pipelines work,
+> you have to change the value to one of the options
+> listed [here](https://argoproj.github.io/argo-workflows/workflow-executors/#workflow-executors)
+> , preferably `k8sapi`.&#x20;
+>
+> This change has to be made by editing the `containerRuntimeExecutor` property
+> of the `ConfigMap` corresponding to the workflow controller. Run the following
+> commands to first know what config map to change and then to edit it to reflect
+> your new value.
+>
+> ```
+> kubectl get configmap -n kubeflow
+> kubectl edit configmap CONFIGMAP_NAME -n kubeflow
+> # This opens up an editor that can be used to make the change.
+> ```
+{% endtab %}
+{% endtabs %}
+
+{% hint style="info" %}
+If one or more of the deployments are not in the `Running` state, try increasing
+the number of nodes in your cluster.
+{% endhint %}
+
+{% hint style="warning" %}
+If you are doing a manual install of the Kubeflow Pipelines, make sure that the
+service name for the ML pipeline is exactly `ml-pipeline` . This will ensure
```

Comment:
`service name for the ML pipeline` seems a little unclear to me at least, is there some better way to describe this?

Reply:
Its even unclear to me, you mind suggesting a formulation @schustmi

Reply:
If you're installing Kubeflow Pipelines manually, make sure the Kubernetes service is called exactly `ml-pipeline`. This will ensure that ZenML can connect to your Kubeflow Pipelines deployment.

Maybe something like this?